src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.php

```diff
@@ -21,6 +21,7 @@
 
 return static function (ContainerConfigurator $container) {
     $container->services()
+        // remove this in Symfony 7.0, the "@required" annotation will be removed
```

Comment:
`@deprecated since Symfony 6.3, to be removed in 7.0` to help with grepping when preparing 7.0